demo.txt

```diff
@@ -1 +1,3 @@
-Initial Content Adding more Content Adding some Content from test Branch
```

Comment:
Good work!  In future, you may want to review your document structure or use a formatting extension to you're editor to reduce the pull request overhead.  Thank you for addressing the issue so quickly.

Reply:
That's good advice, I'm looking into switching from ed to vscode as a result!